core/src/main/java/net/opentsdb/query/BaseQueryNodeConfig.java

```diff
@@ -14,18 +14,22 @@
 // limitations under the License.
 package net.opentsdb.query;
 
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
toss out the wildcards and leave the regular imports please. It just makes it a little cleaner to read in TSDB code so we know what's coming.